yaml/core/of-builder-dep.yml

```diff
@@ -21,7 +21,7 @@ spec:
             secretName: payload-secret
       containers:
       - name: of-builder
-        image: openfaas/of-builder:0.6.0
+        image: openfaas/of-builder:0.6.1
```

Comment:
This version should be mentioned in the README and deploy_swarm.sh, please can you sort this out? I'll push up the new image in time for your PR this morning.